lib/oli_web/common/select_timezone.ex

```diff
@@ -0,0 +1,27 @@
+defmodule OliWeb.Common.SelectTimezone do
+  use Phoenix.Component
+
+  import Phoenix.HTML.Form
+
+  alias Oli.Predefined
+  alias OliWeb.Router.Helpers, as: Routes
+
+  def render(assigns) do
+    ~H"""
+      <script>
+        function submitForm(){
+          const relativePath = window.location.pathname+window.location.search;
+          $('#hidden-redirect-to').val(relativePath);
```

Comment:
This redirect mechanism seems to work great for most pages. I did however run into one instance where is doesn't, and that is on the Create Section page for open and free which has a `source_id` param `https://localhost/admin/open_and_free/new?source_id=publication%3A3`. We'll want to make sure the hidden redirect to path includes any parameters that were in the original url

Reply:
Could you detail a little bit more what you see when updating the timezone for this case? This scenario works correctly in my local, and the search params are included in the refreshed url because of `window.location.search`.

Reply:
This seems to be resolved with the latest changes. I think it may have been related to the empty "Select timezone" timezone selection issue